src/components/OrderHistory.vue

```diff
@@ -2,9 +2,9 @@
   <div class="max-w-2xl m-auto text-center bg-white">
     <div class="border p-10 border-gray-200 shadow rounded">
       <h3 class="text-4xl mb-8 text-gray-700 tracking-wider font-thin">Order History</h3>
-      <div class="divide-y divide-gray-400 w-48 m-auto">
-        <div v-for="order in userOrders" v-bind:key="order.id" class="text-gray-700 pt-2">
-            {{ order.attributes.status }} {{ order.attributes.delivery_date }}
+      <div class="divide-y divide-gray-400 w-56 m-auto">
+        <div v-for="order in userOrders" v-bind:key="order.id" class="link-gray pt-2">
+          <OrderRow v-bind:order="order"></OrderRow>
```

Comment:
I abstracted this out so that each component would have an individual slug that I could pass to the routeer